### PR TITLE
Added Liveinternet (liveinternet.ru) analytics service

### PR DESCRIPTION
**Pull request by kolen**
Vintage web 1.0-era Russian web analytics still widely used on Russian websites, including vk.com and most mail.ru services.

http://www.liveinternet.ru/rating/

Detect by javascript/image code. Seen modified versions of such code on popular sites. vk.com invokes it dynamically from its ``/js/al/common.js`` instead of image, so added ``"script"`` detection rule as exception.

### CODE REVIEW

**Comment by kolen** [comment]
Fixed quotes, added more detail to javascript counter regex (now it matches open ``<script>`` tag and ``src`` within it, with not more than 128 characters between.

Sample ``<script>`` code of liveinternet counters on various sites:

```
<script language="javascript" type="text/javascript">
//<![CDATA[
new Image().src = "//counter.yadro.ru/hit;mail-splash/pc?r"+
escape(document.referrer)+((typeof(screen)=="undefined")?"":
";s"+screen.width+"*"+screen.height+"*"+(screen.colorDepth?
screen.colorDepth:screen.pixelDepth))+";u"+escape(document.URL)+
";"+Math.random();
//]]>
</script>

<script>
(function() {
document.getElementById('li_button').innerHTML = "<img src='//counter.yadro.ru/hit;avito?t26.1;r" + escape(document.referrer) + ((typeof(screen)=="undefined")?"":";s"+screen.width+"*"+screen.height+"*"+(screen.colorDepth?screen.colorDepth:screen.pixelDepth)) + ";u" + escape(document.URL) + ";" + Math.random() + "' border=0 width=88 height=15 alt='' title='LiveInternet: показано число посетителей за сегодня'>";
})();
</script>

<!--LiveInternet counter-->
<script type="text/javascript"><!--
new Image().src = "//counter.yadro.ru/hit;RBCRU?r"+
escape(document.referrer)+((typeof(screen)=="undefined")?"":
";s"+screen.width+"*"+screen.height+"*"+(screen.colorDepth?
        screen.colorDepth:screen.pixelDepth))+";u"+escape(document.URL)+
";"+Math.random();//--></script>
<!--/LiveInternet-->

<!--LiveInternet counter--><script type="text/javascript"><!--
new Image().src = "//counter.yadro.ru/hit;kp/reg/spb?r"+
escape(document.referrer)+((typeof(screen)=="undefined")?"":
";s"+screen.width+"*"+screen.height+"*"+(screen.colorDepth?
screen.colorDepth:screen.pixelDepth))+";u"+escape(document.URL)+
";h"+escape(document.title.substring(0,80))+
";"+Math.random();//--></script><!--/LiveInternet-->
```

File: src/apps.json [hunk 1]
```diff
@@ -3285,6 +3285,19 @@
...
 		},
+		"Liveinternet": {
+			"cats": [
+				10
+			],
+			"html": [
+				"<script[^<>]*>[^]{0,128}?src\\s*=\\s*['\"]//counter\\.yadro\\.ru/hit(?:;\\S+)?\\?(?:t\\d+\\.\\d+;)?r",
+				"<!--LiveInternet counter-->",
+				"<!--/LiveInternet-->",
+				"<a href=\"http://www.liveinternet.ru/click\""
```
Comment (AliasIO) [inline src/apps.json]: This matches any page that links to this URL which is probably not right.

File: src/apps.json [hunk 2]
```diff
@@ -3285,6 +3285,19 @@
 			"script": "livefyre_init\\.js",
 			"website": "livefyre.com"
 		},
+		"Liveinternet": {
+			"cats": [
+				10
+			],
+			"html": [
+				"<script[^<>]*>[^]{0,128}?src\\s*=\\s*['\"]//counter\\.yadro\\.ru/hit(?:;\\S+)?\\?(?:t\\d+\\.\\d+;)?r",
```
Comment (AliasIO) [inline src/apps.json]: Why not use the `script` field for this?

Comment (kolen) [inline src/apps.json]: @AliasIO ``script`` matches ``src`` attribute, not element content. Or maybe it matches both?

File: src/apps.json (hunk 1 above)
Comment (kolen) [inline src/apps.json]: @AliasIO This URL counts "clicks on counter image", I can't imagine case when page will contain link to click counting page of Liveinternet analytics but will not have analytics code.

**Comment by AliasIO** [comment]
Thanks. :)

**Pull request merged**